src/Service/VendorService/EReolenGlobal/EReolenGlobalVendorService.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * @file
+ * Service for updating data from 'eReolen Global / overdrive.com''.
+ */
+
+namespace App\Service\VendorService\EReolenGlobal;
```

Comment:
Don't known if it f...s with the coding style but eReolen normally named `eReolenGlobal`.

Reply:
Did consider this. Decided coding style takes preference.

Reply:
PSR1: _"Class names MUST be declared in StudlyCaps."_ (https://www.php-fig.org/psr/psr-1/#3-namespace-and-class-names)